packages/babel-plugin/src/utils/state-manager.js

```diff
@@ -260,12 +279,40 @@ export default class StateManager {
     this.styleVarsToKeep.add(memberExpression);
   }
 }
-
+// a function generate Regex to match the path of files with aliases
+function generateAliasRegex(alias: string) {
```

Comment:
```suggestion
function generateAliasRegex(alias: string): RegExp {
```

Return types are required in Flow.